apps/etterlatte-behandling/src/main/kotlin/behandling/omregning/OmregningService.kt

```diff
@@ -39,11 +40,17 @@ class OmregningService(
             }
 
             behandlingService.hentAapenOmregning(request.sakId)?.let {
+                val oppgave = oppgaveService.hentOppgaverForReferanse(it.id.toString()).single()
+                if (oppgave.saksbehandler?.navn == Fagsaksystem.EY.navn) {
+                    oppgaveService.fjernSaksbehandler(oppgave.id)
```

Comment:
Ønsker ikke å gjøre dette for alle omregninger og når det gjøres så skal ikke behandlingen avbrytes.
Må vel ha en if INNTEKTSENRING og en return